Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -200,11 +202,15 @@ define([
      * @param {Number} x The X coordinate of the tile for which to request geometry.
      * @param {Number} y The Y coordinate of the tile for which to request geometry.
      * @param {Number} level The level of the tile for which to request geometry.
+     * @param {Boolean} [throttleRequests=true] True if the number of simultaneous requests should be limited,
+     *                  or false if the request should be initiated regardless of the number of requests
+     *                  already in progress.
+     * @param {Number} [distance] The distance of the tile from the camera, used to prioritize requests.
      * @returns {Promise.<TerrainData>|undefined} A promise for the requested geometry.  If this method
      *          returns undefined instead of a promise, it is an indication that too many requests are already
      *          pending and the request will be retried later.
      */
-    ArcGisImageServerTerrainProvider.prototype.requestTileGeometry = function(x, y, level) {
+    ArcGisImageServerTerrainProvider.prototype.requestTileGeometry = function(x, y, level, throttleRequests, distance) {
```

Comment:
I need to look at the rest of the code, but I wonder if `throttleRequests, distance` should be replaced with something like a single `request` object that doesn't throttle when `undefined`, and has a function to return the `priority` (`distance`).  This would also scale nicely to segmenting 3D Tiles vs. terrain vs. imagery, etc.

At the same time, we don't want to make request attempts too heavily if they are going to be made a lot because of throttling (but there's no concern for the ones that result in an actual request).